### PR TITLE
Init task envs variables

### PR DESCRIPTION
**Pull request by geaaru**
Just a fast implementation. Unfortunately inside task_info we haven't pipeline id.

Do you think that could be correct check if a task is inside a pipeline?

Ideas for other environment variables to set? Maybe RootTask?

### CODE REVIEW

File: pkg/tasks/executors/context.go [hunk 1]
```diff
@@ -119,6 +122,9 @@ func (ctx *ExecutorContext) ResolveArtefactsMounts(m ArtefactMapping, i Instruct
 	i.AddMount(artefactdir + ":" + ctx.ContainerPath(artefact_path))
 	i.AddMount(storagedir + ":" + ctx.ContainerPath(storage_path))
 
+	ctx.TargetArtefactDir = ctx.ContainerPath(artefact_path)
+	ctx.TargetStorageDir = ctx.ContainerPath(artefact_path)
```
Comment (mudler) [inline pkg/tasks/executors/context.go]: shouldn't this be ```storage_path``` ?

File: pkg/tasks/executors/context.go [hunk 2]
```diff
@@ -139,6 +145,9 @@ func (ctx *ExecutorContext) ResolveArtefacts(m ArtefactMapping) ArtefactMapping
 	artefactdir = ctx.ContainerPath(artefact_path)
 	storagedir = ctx.ContainerPath(storage_path)
 
+	ctx.TargetArtefactDir = ctx.ContainerPath(artefact_path)
+	ctx.TargetStorageDir = ctx.ContainerPath(artefact_path)
```
Comment (mudler) [inline pkg/tasks/executors/context.go]: Same as above

**Comment by mudler** [comment]
Root task sounds something we would need, maybe also the task type?  (in case the task needs to do different things depending on where it's running) Pretty sure this list would expand over with time.  Thank you!

**Comment by mudler** [comment]
P.S. the PR needs to be opened against develop (it's not setted up as the default branch still)

File: pkg/tasks/executors/context.go (hunk 1 above)
Comment (geaaru) [reply pkg/tasks/executors/context.go]: yes, my fault

File: pkg/tasks/executors/context.go (hunk 2 above)
Comment (geaaru) [reply pkg/tasks/executors/context.go]: yes, cut&paste error.

**Pull request closed without merging**